chainer/functions/pooling/roi_max_align_2d.py

```diff
@@ -127,16 +128,19 @@ def forward_cpu(self, inputs):
                 y = roi_start_h + ph * bin_size_h + \
                     (iy + .5) * bin_size_h / roi_bin_grid_h
                 ix = 0
+                y, y_low, y_high = _get_bounds(y, height)
+                if y is None or y_low is None or y_high is None:
+                    continue
                 while ix < roi_bin_grid_w:
```

Comment:
`for`

Reply:
this PR is for `roi_average_align_2d`.
see https:/github.com/chainer/chainer/pull/5635 for `roi_max_align_2d.py`.